src/credentials/FabricTable.h

```diff
@@ -1093,12 +1091,12 @@ class DLL_EXPORT FabricTable
      * @param[in] fabricIndex - Fabric Index for which to produce the response.
      * @param[in] clientChallenge - Client-provided challenge.
      * @param[in] attestationChallenge - Attestation challenge from the secure session.
-     * @param[inout] outResponse - Reference to a pre-allocated response object that will be populated on success.
+     * @param[out] outResponse - Reference to a pre-allocated response object that will be populated on success.
      * @retval CHIP_NO_ERROR on success
      * @retval CHIP_ERROR_INVALID_ARGUMENT if the fabricIndex or clientChallenge is incorrectly formatted.
      */
-    CHIP_ERROR SignVIDVerificationRequest(FabricIndex fabricIndex, const ByteSpan & clientChallenge,
```

Comment:
Just curious: why all these changes to pass 2-word ByteSpans by value?

Reply:
Follow-up from https://github.com/project-chip/connectedhomeip/pull/38329#discussion_r2042350558, to save flash.